src/style.css

```diff
@@ -2,2 +1,135 @@
+html,
+body,
+div,
+span,
+applet,
+object,
+iframe,
+h1,
+h2,
+h3,
+h4,
+h5,
+h6,
+p,
+blockquote,
+pre,
+a,
+abbr,
+acronym,
+address,
+big,
+cite,
+code,
+del,
+dfn,
+em,
+font,
+img,
+ins,
+kbd,
+q,
+s,
+samp,
+small,
+strike,
+strong,
+sub,
+sup,
+tt,
+var,
+b,
+u,
+i,
+center,
+dl,
+dt,
+dd,
+ol,
+ul,
+li,
+fieldset,
+form,
+label,
+legend,
+table,
+caption,
+tbody,
+tfoot,
+thead,
+tr,
+th,
+td {
   margin: 0;
 }
+
+:root {
+  font-family: Roboto, Arial, sans-serif;
+  box-sizing: border-box;
+  font-size: 12px;
+  font-weight: 500;
+  margin: 0;
```

Comment:
There is a redundant `margin: 0;` declaration here. It's already been set globally on line 63 for all elements, so this line can be removed to clean up the code.